Recruit-Training/Basic-Recruit-Training/person/Person.cpp

```diff
@@ -0,0 +1,67 @@
+#include <iostream>
+#include <string>
+#include "Person.h"
+using namespace std;
+
+
+
+Person::Person()
+{
+    age_ = new int();
+}
+
+Person::~Person()
+{
+    delete age_;
+}
+
+Person::Person(const std::string& name, int age)
+{
+    age_ = new int(age);
+    name_ = name;
+}
+
+void Person::setAge(int newAge)
+{
+    *age_ = newAge;
+}
+
+const std::string& Person::getName() const {
+    return name_;
+}
+
+void Person::setName(const std::string &newName)
```

Comment:
Nitpicking changes the ampersand here should be moved too.

Reply:
done, thank you